src/App.tsx

```diff
@@ -15,71 +17,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
```

Comment:
If the goods array will not have item Jam - you will have bugs.

Better way: `goods[goods.indexOf('Jam')] || goods[0];`